ecosystem/ecosystem.html

```diff
@@ -122,7 +123,7 @@ <h1>
   });
 </script>
 
+<script src="//cdnjs.cloudflare.com/ajax/libs/list.js/1.5.0/list.min.js"></script>
 <script src="//cdnjs.cloudflare.com/ajax/libs/numeral.js/2.0.6/numeral.min.js"></script>
 <script id="filter-script" ecosystem="true" star-count-call-date="ecosystemStarCountCallDate" star-count-data="ecosystemStarCountData" src="{{ site.baseurl }}/assets/github-stars.js"></script>
-<script src="//cdnjs.cloudflare.com/ajax/libs/list.js/1.5.0/list.min.js"></script>
-<script list-id="ecosystem-index-cards" display-count="100" pagination="false" src="{{ site.baseurl }}/assets/filter-hub-tags.js"></script>
+<script list-id="ecosystem-index-cards" display-count="10" pagination="true" src="{{ site.baseurl }}/assets/filter-hub-tags.js"></script>
```

Comment:
It looks like you have a merge conflict related to these scripts. Once you address the conflict then this should be good to go.